### PR TITLE
Simplify BedrockTester

### PR DESCRIPTION
**Pull request by tylerkaraszewski**
While doing a lot of testing, dealing with similar but disparate Tester classes in Auth and Bedrock, and dealing with redundant code in both of them, I decide to clean them both up, and make AuthTester a subclass of BedrockTester, such that improvements to BedrockTester don't need to be replicated in AuthTester.

This is the bedrock portion of the change. A separate PR will be made in Server-Expensify for the auth change.

### CODE REVIEW

File: test/lib/BedrockTester.cpp [hunk 1]
```diff
@@ -331,35 +275,35 @@ vector<pair<string,string>> BedrockTester::executeWaitMultiple(vector<SData> req
                         }
                     } else {
                         timeouts++;
-                        if (timeouts > 5) {
-                            //SAUTOLOCK(listLock);
-                            //cout << "Timeout (" << timeouts << ") waiting on socket, will try again." << endl;
-                        }
-                        if (timeouts == 60) {
+                        if (timeouts == 600) {
```
Comment (mea36) [inline test/lib/BedrockTester.cpp]: You changed this to `600` but use `60` below.  Is there a reason for that?

File: test/lib/BedrockTester.cpp [hunk 2]
```diff
@@ -42,164 +27,113 @@ string BedrockTester::getTempFileName(string prefix) {
...
     }
-    return *writableDB;
 }
 
-// This is sort of convoluted because of the way it was originally built. We can probably just have this always
-// use the member variable db and never open it's own handle. This was added early ob for debugging and should be
-// obsolete.
-string BedrockTester::readDB(const string& query) { return getSQLiteDB().read(query); }
-
-bool BedrockTester::readDB(const string& query, SQResult& result) { return getSQLiteDB().read(query, result); }
```
Comment (mea36) [inline test/lib/BedrockTester.cpp]: Why are you removing this?

Comment (tylerkaraszewski) [reply test/lib/BedrockTester.cpp]: It's unused (it was even marked obsolete in the comment that's removed). There's an equivalent function added in AuthTester where it is used.

File: test/lib/BedrockTester.cpp (hunk 1 above)
Comment (tylerkaraszewski) [reply test/lib/BedrockTester.cpp]: I updated the other one to match.

File: test/lib/BedrockTester.cpp (hunk 2 above)
Comment (tylerkaraszewski) [reply test/lib/BedrockTester.cpp]: Ok, added back.

**Pull request merged**